normandy/base/api/mixins.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.views.decorators.cache import cache_control
+
+
+class CachingMixin:
```

Comment:
Also maybe call this `CachingViewsetMixin` to make it clear what it's a mixin for?